satellite/satellitedb/containment.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"context"
+	"database/sql"
+
+	"storj.io/storj/pkg/audit"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	dbx "storj.io/storj/satellite/satellitedb/dbx"
+)
+
+type containment struct {
+	db *dbx.DB
+}
+
+// Get gets the pending audit by node id
+func (containment *containment) Get(ctx context.Context, id pb.NodeID) (*audit.PendingAudit, error) {
+	if id.IsZero() {
+		return nil, audit.ContainError.New("node ID empty")
+	}
+
+	pending, err := containment.db.Get_PendingAudits_By_NodeId(ctx, dbx.PendingAudits_NodeId(id.Bytes()))
+	if err == sql.ErrNoRows {
+		return nil, audit.ErrContainedNotFound.New(id.String())
+	}
+	if err != nil {
+		return nil, audit.ContainError.Wrap(err)
+	}
+
+	return convertDBPending(pending)
+}
+
+// IncrementPending will create a new pending audit entry, or increase its reverify count if it already exists
+func (containment *containment) IncrementPending(ctx context.Context, pendingAudit *audit.PendingAudit) error {
```

Comment:
I feel like `IncrementPending` should return an int or something that represents the current `reverify_count`. That way we don't have to follow it with a `Get` in order to determine if we've hit the max limit

Reply:
oh, that's a good idea. I would probably need to add a query to get the value here though right?

Reply:
hey @kaloyan-raev I saw you thumbs-up'd Moby's comment, but just wanted to let you know that I decided not to write a new query inside the IncrementPending query, since it would do about the same as a Get anyway. do you think that's okay?

Reply:
I think it is OK.

Reply:
I've seen this as resolved but I wanted to post this comment just for being considered, not for being applied.

In _postgres_ there is the possibility to return an inserted/updated value https://www.postgresql.org/docs/9.5/sql-insert.html however it doesn't look that it can be done with sqlite, so we may not want to use it because it isn't going to work across SQL databases.

I'm not an expert on DB either, so I don't know if that `returning` instruction is bad from the performance point of view.